examples/python/alphabet/alphabet.py

```diff
@@ -26,11 +26,12 @@
 
 
 def application_setup(args):
-    in_host, in_port = wallaroo.tcp_parse_input_addrs(args)[0]
+    in_name, in_host, in_port = wallaroo.tcp_parse_input_addrs(args)[0]
     out_host, out_port = wallaroo.tcp_parse_output_addrs(args)[0]
 
     votes = wallaroo.source("alphabet",
-                       wallaroo.TCPSourceConfig(in_host, in_port, decode_votes))
+                       wallaroo.TCPSourceConfig(in_host, in_port, in_name,
```

Comment:
I think the name should be the first parameter when creating a TCPSourceConfig like this.

Reply:
This will involve changing both the signature for this function, and the order that things are read and translated in `machida.pony`

Reply:
Note that in the pony side, TCPSourceConfig also takes its name as the third arg. I tried to keep things consistent.
I think if we make this change, we should change the pony configs` parameter order as well.

Reply:
I agree.  I must have missed that in the last PR.

Reply:
@nisanharamati let me know if you want to do the change on the pony side TCPSourceConfig as part of this PR or if I should open a separate PR for that

Reply:
@JONBRWN I'll have it as part of this one. I'm almost done with the changes.